### PR TITLE
ci: ignore some changes for tests

### PR DESCRIPTION
**Pull request by eliobischof**

### CODE REVIEW

**Review by fforootd — DISMISSED** [review]

**Comment by eliobischof** [comment]
I tested it successfully here https://github.com/zitadel/zitadel/pull/4154

**Comment by eliobischof** [comment]
why is Test not in the list of checks now? :-D

File: .github/workflows/test-code.yml [hunk 1]
```diff
@@ -2,6 +2,11 @@ name: ZITADEL PR
 
 on:
   pull_request:
+    paths-ignore:
+      - 'docs/**'
+      - 'guides/**'
+      - 'tools/**'
```
Comment (livio-a) [inline .github/workflows/test-code.yml:8]: @adlerhurst changes in this folder have an impact on the build / code, right?

Comment (adlerhurst) [reply .github/workflows/test-code.yml:8]: yes. The binaries installed from this mod file may change behaviours

Comment (eliobischof) [reply .github/workflows/test-code.yml]: not ignoring the directory anymore https://github.com/zitadel/zitadel/pull/4152/commits/2858f5c23989384e082cd8f9255f133027612fed

**Review by livio-a — APPROVED** [review]

**Pull request merged**